### PR TITLE
Updated Documentation for GCBM REST API

### PR DESCRIPTION
**Pull request by Saurabh-Suchak**
# Pull Request Template

## Description

Since a new GCBM run has been developed, this PR contains a new readme file containing the steps to run GCBM simulation locally. 

Fixes  the old documentation with updated instructions.

## Type of change

Please delete options that are not relevant.

- [x] This change requires a documentation update

## How Has This Been Tested?

The steps provided in the readme are executed are multiple times and verified.

### CODE REVIEW

File: local/rest_api_gcbm/README.md [hunk 1]
```diff
@@ -1,38 +1,88 @@
...
-|  **\gcbm\list**  | This endpoint retrieves the complete list of simulations that are created using /new.  | `GET`
+|   **/help/all**    | This endpoint produces a help message with information on all options for moja.CLI | `GET`
+|   **/help/arg**    |  This endpoint produces a help message with information on option arg for moja.CLI. | `GET`
+|   **/gcbm/new**    |  This endpoint creates a new directory to store input and output files of the simulation. Parameter to be passed in the body is the title of the new simulation or default value simulation will be used.   | `POST` |
+|  **/gcbm/upload**  | This endpoint is used to upload the input files: config_files, input and database. Remember to upload the files in the body from the GCBM folder which is available in the root of this repository for setup and use. A directory /input will be created to store the specified files. | `POST` |
+| **/gcbm/dynamic**  |  This endpoint runs simulation in a thread and generates an output zip file in /output directory. The process may take a while. Parameter is the title of the simulation. | `POST` |
```
Comment (aornugent) [inline local/rest_api_gcbm/README.md:36]: Can you please file an issue to rename this endpoint `/gcbm/run`? I don't know why it's called dynamic

File: local/rest_api_gcbm/README.md [hunk 2]
```diff
@@ -1,38 +1,88 @@
...
 2. The endpoints can be interacted with using `cURL`. 
 curl is used in command lines or scripts to transfer data. Find out more about curl [here](https://curl.se/). Commands using cURL can be found [here](curl.md)
+<br/>
+To run the endpoints and start a new GCBM simulation : 
+
+ Unzip `GCBM_New_Demo_Run` and open a terminal inside the folder and run the following curl commands:
+ 
+ 1. Create a new simulation (the default title of the simulation is `simulation`, here the title `run4` is used): 
+     ```
+        curl -d "title=run4" -X POST http://localhost:8080/gcbm/new
```
Comment (aornugent) [inline local/rest_api_gcbm/README.md:56]: The title can be anything eg. "example-run"

**Review by aornugent — APPROVED** [review]

**Comment by padmajabhol** [comment]
Hey @Saurabh-Suchak is it possible for you to change the PR to a meaningful name? "Documentation for GCBM REST API" or something on that line?

**Review by padmajabhol — CHANGES_REQUESTED** [review]
@Namyalg had worked on the the endpoint that lets us change the table schema for the data base.
Please refer to https://github.com/Namyalg/gcbm-table-schema to understand how it works and we can interact with the endpoint. I'm sure we need it to go along with the entire documentation.

File: local/rest_api_gcbm/README.md (hunk 1 above)
Comment (Saurabh-Suchak) [reply local/rest_api_gcbm/README.md:36]: Will do it.

**Comment by Saurabh-Suchak** [comment]
> @Namyalg had worked on the the endpoint that lets us change the table schema for the data base. Please refer to https://github.com/Namyalg/gcbm-table-schema to understand how it works and we can interact with the endpoint. I'm sure we need it to go along with the entire documentation.

Sure, I will update the documentation as per the requirement

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**